### PR TITLE
small fix to precommit (#18126)

### PR DESCRIPTION
**Pull request by garyverhaegen-da**

### CODE REVIEW

File: dev-env/bin/dade-check-main-only-files [hunk 1]
```diff
@@ -6,11 +6,11 @@
 # on a PR branch which touches those files.
 #
 
-DADE_SKIP_MAIN_ONLY_CHECK="${DADE_SKIP_MAIN_ONLY_CHECK:-$(git log -1 --format=%b | tr -d ' ' | grep '^run-all-tests:true$')}"
+DADE_SKIP_MAIN_ONLY_CHECK="${DADE_SKIP_MAIN_ONLY_CHECK:-$(git log -1 --format=%b | tr -d ' ' | grep '^run-all-tests: *true$')}"
```
Comment (akrmn) [inline dev-env/bin/dade-check-main-only-files:9]: huh shouldn't `tr -d ' '` have removed all the spaces?

Comment (samuel-williams-da) [reply dev-env/bin/dade-check-main-only-files:9]: I would guess that this fix doesn't work then :/

Comment (garyverhaegen-da) [reply dev-env/bin/dade-check-main-only-files:9]: Ah, sorry: I don't use precommit hooks, I assumed you had tested it when you approved the merge to main.

Comment (samuel-williams-da) [reply dev-env/bin/dade-check-main-only-files:9]: Ah oops, I did not, the fix seemed so simple i assumed it would work :/

Comment (samuel-williams-da) [reply dev-env/bin/dade-check-main-only-files:9]: Lets disable automerge for now to debug this

Comment (garyverhaegen-da) [reply dev-env/bin/dade-check-main-only-files:9]: I've done a bit of local testing and it seems to work as expected for me, both with and without this change (i.e. confirming what @akrmn said).

**Pull request closed without merging**